src/test/java/io/aiven/kafka/connect/transforms/ExtractTimestampConfigTest.java

```diff
@@ -51,4 +51,40 @@ void definedFieldName() {
         final ExtractTimestampConfig config = new ExtractTimestampConfig(props);
         assertEquals("test", config.fieldName());
     }
+
+    @Test
+    void emptyTimestampResolution() {
+        final var props = new HashMap<>();
+        props.put("field.name", "test");
+        final var config = new ExtractTimestampConfig(props);
+        assertEquals(ExtractTimestampConfig.TimestampResolution.MILLISECONDS, config.timestampResolution());
+    }
+
+    @Test
+    void definedTimestampResolution() {
```

Comment:
[Small thing] Should this test also include explicitly defined `MILLISECONDS` resolution?

Reply:
will add